bindingcollectionadapter/src/main/java/me/tatarka/bindingcollectionadapter2/itembindings/OnItemBindClass.java

```diff
@@ -18,18 +21,30 @@
  */
 public class OnItemBindClass<T> implements OnItemBind<T> {
 
-    private final SimpleArrayMap<Class<? extends T>, int[]> itemBindingMap;
+    private final SimpleArrayMap<Class<? extends T>, Object> itemBindingMap;
+    // to support ordinal mapping {@see OnItemBindClassTest#selectsBasedOnClassAndSubclass()}
```

Comment:
Replacement matches the existing behavior. I'd be tempted to say fail on duplicate but I don't know if that would break some usecase.